### DISCUSSION TITLE
THANKS a lot for creating Deta

### DISCUSSION THREAD

**Discussion by gloriamacia** [deta/cloud-docs#603] [Show and tell]
Dear team, 

https://ghana.deta.dev/ - to be finished. 

Deta will be soon helping covid19 vaccination uptake in Ghana as part of an NGO program. 

I would like to thank you for making Deta so great and free.

**Comment by abdelhai** [comment]
thanks for using Deta! 🥰

**Comment by pydev369** [comment]
Which Tech stack used? Any open-source packages used ?
